.github/workflows/create_draft_release.yml

```diff
@@ -105,3 +105,16 @@ jobs:
         working-directory: ${{steps.assets.outputs.artifacts_path}}
         run: |
           dotnet nuget push "*.${{steps.versions.outputs.full_version}}*.nupkg" --api-key ${{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json
+
+      - name: Prepare debug symbols for publishing
+        shell: bash
+        run: |
+          mkdir debug_symbols
+          tar zxcf linux-native-symbols.tar.gz -C debug_symbols
+
+      - name: 'Push debug symbols to datadog'
+        uses: ./.github/actions/publish-debug-symbols
+        with:
+          symbols_folder: "debug_symbols"
+          preprod_key: "${{ secrets.DD_PREPROD_API_KEY }}"
+          prod_key: "${{ secrets.DD_API_KEY }}"
```

Comment:
Question: is it better to do this _before_ or _after_ we publish to NuGet/create the release? 🤔 

If before, then if there's any issues the good news is we can correct them first, and we can't create releases _without_ symbols. The bad news is it would block the release if there's a problem, or if datadog is down or something 🤔 

I think I'm inclined to go with your approach, especially as we can also do this manually using the other action if there _is_ a problem later 🤔

Reply:
👍